spec/automation/unit/method_validation/task_finished_spec.rb

```diff
@@ -1,6 +1,8 @@
 describe "task_finished_status" do
   include Spec::Support::AutomationHelper
 
+  before { NotificationType.seed }
```

Comment:
Use the factory to create a `NotificationType` instance and move the call into the single test that needs it instead of doing a full seeding.